lib/web/apiserver.go

```diff
@@ -2337,7 +2337,7 @@ func (h *Handler) hostCredentials(w http.ResponseWriter, r *http.Request, p http
 		return nil, trace.Wrap(err)
 	}
 
-	return auth.LegacyCertsFromProto(certs), nil
+	return certs, nil
```

Comment:
```suggestion
	// Teleport 8 clients are still expecting the legacy JSON format.
	// Teleport 9 clients handle both legacy and new.
	// TODO(zmb3) return certs directly in Teleport 10
	return auth.LegacyCertsFromProto(certs), nil
```

Reply:
I don't understand what you're suggesting.